src/main/java/com/epam/lowcost/controller/TicketController.java

```diff
@@ -80,4 +79,11 @@ public String deleteTicket(@RequestParam long id, Model model) {
     }
 
 
+    @GetMapping(value = "/my-tickets")
```

Comment:
Possibly '/my' can be enough. or '/self'. Up to you. Double word 'ticket' is unnecessary

Reply:
Changed to /tickets/self